RecoLocalCalo/EcalRecAlgos/src/EigenMatrixTypes.cc

```diff
@@ -0,0 +1,2 @@
+#include "RecoLocalCalo/EcalRecAlgos/interface/EigenMatrixTypes.h"
```

Comment:
is this file actually needed? EigenMatrixTypes.h contains only typedefs and constexpr.